src/inputs-controller.ts

```diff
@@ -507,17 +590,19 @@ export class InputsController {
   }
 
   /**
-   * Retrieves the configuration object for a keyboard layout based on its identifier.
-   *
-   * @param id The identifier string of the keyboard layout.
-   * @returns InterfaceConfig The configuration object corresponding to the identified keyboard layout.
+   * Retrieves the configuration object for a given keyboard layout.
+   * @param layout The {@linkcode KeyboardLayout} identifier string of the keyboard layout.
+   * @returns {@linkcode KeyboardInterfaceConfig} for this layout.
```

Comment:
```suggestion
   * @returns The {@linkcode KeyboardInterfaceConfig} for this layout.
```
VSCode doesn't like when you start the `@returns` line with `{@linkcode`:
<img width="734" height="140" alt="image" src="https://github.com/user-attachments/assets/00f0fd94-18f8-4398-9bcb-ac262be4e2f4" />